ops-devnet/devnet-up.sh

```diff
@@ -97,14 +96,16 @@ fi
 
 L2OO_ADDRESS="0x6900000000000000000000000000000000000004"
 COLOSSEUM_ADDRESS="0x690000000000000000000000000000000000000D"
+VP_ADDRESS=""
```

Comment:
We need to set this address, but I'm not sure what address to set it so I left it empty.